coredb-operator/src/statefulset.rs

```diff
@@ -96,8 +54,55 @@ fn stateful_set_from_cdb(cdb: &CoreDB) -> StatefulSet {
             },
             template: PodTemplateSpec {
                 spec: Some(PodSpec {
-                    containers: vec![postgres_container],
-                    init_containers: Option::from(vec![postgres_init_container]),
+                    containers: vec![
+                        // This container for running postgresql
+                        Container {
+                            env: postgres_env.clone(),
+                            security_context: Some(SecurityContext {
+                                run_as_user: Some(cdb.spec.uid.clone() as i64),
+                                allow_privilege_escalation: Some(false),
+                                ..SecurityContext::default()
+                            }),
+                            name: "postgres".to_string(),
+                            image: Some(cdb.spec.image.clone()),
+                            ports: Some(vec![ContainerPort {
+                                container_port: 5432,
+                                ..ContainerPort::default()
+                            }]),
+                            volume_mounts: postgres_volume_mounts.clone(),
+                            readiness_probe: Some(Probe {
+                                exec: Some(ExecAction {
+                                    command: Some(vec![String::from("pg_isready")]),
+                                }),
+                                initial_delay_seconds: Some(3),
+                                ..Probe::default()
+                            }),
```

Comment:
this is new logic to add a readiness probe, using the postgres feature "pg_isready"